pkgs/applications/video/mpv/default.nix

```
@@ -154,18 +160,22 @@ in stdenv'.mkDerivation (finalAttrs: {
     (lib.mesonEnable "videotoolbox-pl" vulkanSupport)
   ];
 
-  mesonAutoFeatures = "auto";
-
   nativeBuildInputs = [
     addOpenGLRunpath
     docutils # for rst2man
     meson
     ninja
     pkg-config
   ]
-  ++ lib.optionals stdenv.isDarwin [ xcbuild.xcrun rcodesign ]
-  ++ lib.optionals swiftSupport [ swift ]
-  ++ lib.optionals waylandSupport [ wayland-scanner ];
+  ++ lib.optionals waylandSupport [
+    wayland-scanner
+  ]
+  ++ lib.optionals swiftSupport [
+    swift
+  ]
+  ++ lib.optionals stdenv.isDarwin [
+    xcbuild.xcrun
+  ];
```

`darwin.sigtool` is required again without the patch. It was forwarded via `inherit (darwin) sigtool;` in `all-packages.nix`:
```suggestion
  ++ lib.optionals stdenv.isDarwin [
    sigtool
    xcbuild.xcrun
  ];
```